orchestrators/ProcurementOperations/src/Services/NullVendorRecommendationLlm.php

```
@@ -0,0 +1,21 @@
+
+use Nexus\ProcurementOperations\Contracts\VendorRecommendationLlmInterface;
+use Nexus\ProcurementOperations\DTOs\VendorRecommendation\VendorRecommendationRequest;
+
+final readonly class NullVendorRecommendationLlm implements VendorRecommendationLlmInterface
+{
+    /**
+     * @param list<\Nexus\ProcurementOperations\DTOs\VendorRecommendation\VendorRecommendationScoredCandidate> $candidates
+     *
+     * @return array<string, array<string, mixed>>
+     */
+    public function enrich(VendorRecommendationRequest $request, array $candidates): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)